karaf/features/src/main/history/dependencies.xml

```diff
@@ -2,46 +2,102 @@
+        <bundle>mvn:org.apache.brooklyn/brooklyn-camp/0.9.0-SNAPSHOT</bundle>
         <bundle>mvn:org.apache.brooklyn/brooklyn-commands/0.9.0-SNAPSHOT</bundle>
         <bundle>mvn:org.apache.brooklyn/brooklyn-core/0.9.0-SNAPSHOT</bundle>
+        <bundle>mvn:org.apache.brooklyn/brooklyn-jsgui/0.9.0-SNAPSHOT/war</bundle>
         <bundle>mvn:org.apache.brooklyn/brooklyn-logback-includes/0.9.0-SNAPSHOT</bundle>
+        <bundle>mvn:org.apache.brooklyn/brooklyn-rest-api/0.9.0-SNAPSHOT</bundle>
+        <bundle>mvn:org.apache.brooklyn/brooklyn-rest-server/0.9.0-SNAPSHOT</bundle>
         <bundle>mvn:org.apache.brooklyn/brooklyn-rt-osgi/0.9.0-SNAPSHOT</bundle>
         <bundle>mvn:org.apache.brooklyn/brooklyn-utils-common/0.9.0-SNAPSHOT</bundle>
         <bundle>mvn:org.apache.brooklyn/brooklyn-utils-common/0.9.0-SNAPSHOT</bundle>
         <bundle>mvn:org.apache.brooklyn/brooklyn-utils-groovy/0.9.0-SNAPSHOT</bundle>
+        <bundle>mvn:org.apache.brooklyn/brooklyn-utils-rest-swagger/0.9.0-SNAPSHOT</bundle>
+        <bundle>mvn:org.apache.commons/commons-compress/1.4</bundle>
+        <bundle>mvn:org.apache.commons/commons-lang3/3.1</bundle>
+        <bundle>mvn:org.apache.commons/commons-lang3/3.1</bundle>
+        <bundle>mvn:org.apache.commons/commons-lang3/3.1</bundle>
         <bundle>mvn:org.apache.commons/commons-lang3/3.1</bundle>
         <bundle>mvn:org.apache.httpcomponents/httpclient-osgi/4.4.1</bundle>
         <bundle>mvn:org.apache.httpcomponents/httpcore-osgi/4.4.1</bundle>
         <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jzlib/1.1.3_2</bundle>
+        <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.reflections/0.9.9_1</bundle>
         <bundle>mvn:org.bouncycastle/bcpkix-jdk15on/1.49</bundle>
         <bundle>mvn:org.bouncycastle/bcprov-ext-jdk15on/1.49</bundle>
         <bundle>mvn:org.codehaus.groovy/groovy-all/2.3.7</bundle>
         <bundle>mvn:org.codehaus.jackson/jackson-core-asl/1.9.13</bundle>
+        <bundle>mvn:org.codehaus.jackson/jackson-core-asl/1.9.13</bundle>
+        <bundle>mvn:org.codehaus.jackson/jackson-core-asl/1.9.13</bundle>
         <bundle>mvn:org.codehaus.jackson/jackson-jaxrs/1.9.13</bundle>
         <bundle>mvn:org.codehaus.jackson/jackson-mapper-asl/1.9.13</bundle>
+        <bundle>mvn:org.codehaus.jackson/jackson-mapper-asl/1.9.13</bundle>
         <bundle>mvn:org.freemarker/freemarker/2.3.22</bundle>
+        <bundle>mvn:org.jvnet.mimepull/mimepull/1.9.3</bundle>
         <bundle>mvn:org.slf4j/jul-to-slf4j/1.6.6</bundle>
         <bundle>mvn:org.yaml/snakeyaml/1.11</bundle>
         <bundle>wrap:mvn:com.google.http-client/google-http-client/1.18.0-rc</bundle>
         <bundle>wrap:mvn:com.maxmind.geoip2/geoip2/0.8.1</bundle>
-        <bundle>wrap:mvn:org.reflections/reflections/0.9.9-RC1</bundle>
+        <bundle>wrap:mvn:javax.validation/validation-api/1.1.0.Final</bundle>
+        <bundle>wrap:mvn:org.tukaani/xz/1.4</bundle>
```

Comment:
It is needed for strict dependency control, if wanted. There are some flags in the pom that control if this is generated, or whether to warn/fail if dependencies change. Just ignore it for the moment.